internal/registry/config.go

```diff
@@ -114,3 +154,24 @@ func isPodReady(pod corev1.Pod) bool {
 
 	return false
 }
+
+func getServedDockerRegistry(ctx context.Context, c dynamic.Interface) (*DockerRegistry, error) {
+	list, err := c.Resource(DockerRegistryGVR).List(ctx, metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	for _, item := range list.Items {
+		var dockerRegistry DockerRegistry
+		err = kube.FromUnstructured(&item, &dockerRegistry)
+		if err != nil {
+			return nil, err
+		}
+
+		if dockerRegistry.Status.Served == "True" {
```

Comment:
Why you look on `Served`? IMO better could be `State == "Ready"`.

Reply:
done

Reply:
good point, thanks